docs/design/noc-root-cert-design.md

```diff
@@ -15,48 +18,64 @@ To distinguesh NOC root certificates from others, an `isNOC` boolean field will
 ## Transactions
 
 ### 1. ADD_NOC_X509_ROOT_CERTIFICATE
-This transaction adds a NOC Root Certificate owned by the Vendor.
+This transaction adds a NOC root certificate owned by the Vendor.
 
 - Who can send: Vendor account
-  - VID-scoped NOC Root Certificate: The `vid` field in the certificate's subject must be equal to the Vendor account's VID.
 - Validation:
   - The provided certificate must be a root certificate:
     - `Issuer` == `Subject`
     - `Authority Key Identifier` == `Subject Key Identifier`
   - No existing certificate with the same `<Certificate's Issuer>:<Certificate's Serial Number>` combination.
```

Comment:
We need to clarify what of a certificate with the combination Subject:SKID is revoked. Should we be able to re-add it.

Reply:
Mentioned in the REVOKE_NOC_X509_ROOT_CERTIFICATE transaction section